jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -336,31 +330,21 @@ public Image getImageForSettler(EMovableType movableType, EMovableAction action,
 
 	/**
 	 * Gets a map item.
-	 * 
+	 *
 	 * @param movableType
 	 * @param action
 	 * @param material
 	 * @param direction
 	 * @param progress
 	 * @return The item of the map at the given position. Is not null.
 	 */
-	private SettlerImageMapItem getMapItem(EMovableType movableType,
-			EMovableAction action, EMaterialType material, EDirection direction) {
-
-
-		if(movableType == null
-				|| action == null
-				|| material == null
-				|| direction==null) {
+	private SettlerImageMapItem getMapItem(EMovableType movableType, EMovableAction action, EMaterialType material, EDirection direction) {
```

Comment:
@rpolzer: I think your formatter might be configure to a line width of 80. We use a line width of 200 for our formatting.